cmd/mimir/help-all.txt.tmpl

```diff
@@ -1145,6 +1145,12 @@ Usage of ./cmd/mimir/mimir:
     	Per-tenant allowed push request burst size. 0 to disable.
   -distributor.request-rate-limit float
     	Per-tenant push request rate limit in requests per second. 0 to disable.
+  -distributor.retry.base duration
+    	[experimental] Base time to wait before retrying a 429/5xx request. (default 3s)
```

Comment:
well this isn't about waiting. It's about the value of the `Retrey-After` header, right?